Explorer/Assets/DCL/MapRenderer/MapLayers/Atlas/SatelliteAtlas/SatelliteChunkController.cs

```diff
@@ -46,27 +50,31 @@ public void Dispose()
             internalCts?.Dispose();
             internalCts = null;
 
-            if (spriteRenderer)
-                UnityObjectUtils.SafeDestroy(spriteRenderer.gameObject);
+            if (atlasChunk)
+                UnityObjectUtils.SafeDestroy(atlasChunk.gameObject);
         }
 
         public async UniTask LoadImageAsync(Vector2Int chunkId, float chunkWorldSize, CancellationToken ct)
         {
             webRequestAttempts = 0;
             linkedCts = CancellationTokenSource.CreateLinkedTokenSource(internalCts.Token, ct);
-
+            atlasChunk.MainSpriteRenderer.enabled = false;
+            atlasChunk.MainSpriteRenderer.color = initialColor;
             var url = $"{CHUNKS_API}{chunkId.x}%2C{chunkId.y}.jpg";

```

Comment:
Not part of your PR, but I think we can improve this by calling `.ToString()` on `chunkId.x` and `chunckId.y` to avoid some boxing allocations

Reply:
Got it, I missed the %2C lol. Also thanks for taking the time to explain all that, I appreciate it!